hoomd/mpcd/test-py/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ set(TEST_LIST
     stream_slit
     stream_slit_pore
     stream_sphere
+    stream_moving_sphere
```

Comment:
Tests should be named after the classes, so we need to decide how to name and implement the class in Python first, then we can add the test.